includes/helper/class-wp-job-manager-helper.php

```diff
@@ -384,7 +466,7 @@ public function is_product_installed( $product_slug ) {
 	 * @return bool
 	 */
 	public function has_licenced_products() {
-		$product_plugins = $this->get_installed_plugins();
+		$product_plugins = $this->get_installed_plugins( false, false );
```

Comment:
This is another change in implementation. Beforehand this would only care about activated licensed plugins. This is only used to decide whether to force showing the license page, which we want for inactive plugins.